nixos/modules/services/misc/mediatomb.nix

```diff
@@ -36,7 +36,6 @@ let
         <transcode mimetype="video/x-flv" using="vlcmpeg" />
         <transcode mimetype="application/ogg" using="vlcmpeg" />
         <transcode mimetype="audio/ogg" using="ogg2mp3" />
-        <transcode mimetype="audio/x-flac" using="oggflac2raw"/>
```

Comment:
Is the removal of this line intended? If so please split it into its own commit.

Reply:
Actually, this profile does not work with current vlc version, should I make a separated commit anyway?

Reply:
That would be preferable, and make sure to mention `this profile does not work with current vlc version` in the commit message.